config.json

```diff
@@ -1,7 +1,7 @@
 {
   "language": "Swift",
   "active": true,
-  "blurb": "",
+  "blurb": "Swift is a modern open source programming language that runs on macOS, Linux and on Windows using a linux subsystem.",
   "exercises": [
```

Comment:
I'd love to have something a bit more approachable here than where the language runs.

E.g. the blurb for R specifically calls out the powerful data manipulation and data science applications.